tests/Unit/Domain/Test_DomainHelpers.cpp

```diff
@@ -275,7 +277,7 @@ void test_wedge_map_generation_against_domain_helpers(
     double inner_radius, double outer_radius, double inner_sphericity,
     double outer_sphericity, bool use_equiangular_map,
     double x_coord_of_shell_center = 0.0, bool use_half_wedges = false,
-    double aspect_ratio = 1.0) {
+    double aspect_ratio = 1.0, size_t index_polar_axis = 2) {
```

Comment:
This isn't used here. Forgot to pass it on to `test_wedge_map_generation` and `sph_wedge_coordinate_maps`?